cross/arm32_ci_script.sh

```diff
@@ -189,13 +189,13 @@ function cross_build_corefx_with_docker {
         # TODO: For arm, we are going to embed RootFS inside Docker image.
         case $__linuxCodeName in
         trusty)
-            __dockerImage=" microsoft/dotnet-buildtools-prereqs:ubuntu1404_cross_prereqs_v3"
+            __dockerImage=" microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-0cd4667-20172211042239"
```

Comment:
Yes, the SHA was the artifact I was referring to. Sounds fine to me :)

Are we going to publish (somewhere) the name of the images with their highlevel description of changes?